{"level":"error","message":"error error","stack":"Error: error\n    at D:\\tree-cover-loss\\kuyua.tree.cover.loss.API\\index.js:15:9\n    at Layer.handle [as handle_request] (D:\\tree-cover-loss\\kuyua.tree.cover.loss.API\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (D:\\tree-cover-loss\\kuyua.tree.cover.loss.API\\node_modules\\express\\lib\\router\\route.js:149:13)\n    at Route.dispatch (D:\\tree-cover-loss\\kuyua.tree.cover.loss.API\\node_modules\\express\\lib\\router\\route.js:119:3)\n    at Layer.handle [as handle_request] (D:\\tree-cover-loss\\kuyua.tree.cover.loss.API\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at D:\\tree-cover-loss\\kuyua.tree.cover.loss.API\\node_modules\\express\\lib\\router\\index.js:284:15\n    at Function.process_params (D:\\tree-cover-loss\\kuyua.tree.cover.loss.API\\node_modules\\express\\lib\\router\\index.js:346:12)\n    at next (D:\\tree-cover-loss\\kuyua.tree.cover.loss.API\\node_modules\\express\\lib\\router\\index.js:280:10)\n    at D:\\tree-cover-loss\\kuyua.tree.cover.loss.API\\middlewares\\AuthMiddleware.js:10:13\n    at module$contents$ee$apiclient_apiclient.handleAuthResult_ (D:\\tree-cover-loss\\kuyua.tree.cover.loss.API\\node_modules\\@google\\earthengine\\build\\main.js:19389:16)"}
